tests/integration/run_tests.py

```diff
@@ -596,15 +606,6 @@ async def main():
         LOG.debug("=== Login other mock user ===")
         await login(sess, other_test_user, other_test_user_given, other_test_user_family)
 
-        # Test add, modify, validate and release action with submissions
-        LOG.debug("=== Testing actions within submissions ===")
-        submission_folder = {
-            "name": "submission test 1",
-            "description": "submission test folder 1",
-        }
-        submission_folder_id = await post_folder(sess, submission_folder)
-        await test_submissions_work(sess, submission_folder_id)
-
```

Comment:
This seemed like a duplicate test so removed it... unless there's a reason for it to be there?

Reply:
that was there for a purpose to show that objects separation is done for each user

Reply:
reverted this commit so it's back in there